taqueria-vscode-extension-web-ui/src/data-editors/BoolEditor.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { isObject } from '../Helpers';
+
+export const BoolEditor = (
+	{ value, onChange }: { value: any; onChange: (value: { prim: 'True' | 'False' }) => void },
+) => {
+	if (!isObject(value) && !['False', 'True'].includes(value.prim)) {
+		value = {
```

Comment:
Mutating props is not React idiomatic 🤔 What you wanted to achieve here?

Reply:
Fixed, thanks